unfoldingWord/translationCore
# Added collapsible submenu to Group Menu

#### This pull request addresses:

Added collapsible submenu to Group Menu

#### How to test this pull request:

NOTICE: 
There is a data issue in tW, where the entire header is in the quote, and it should be the quote only.
This is a separate issue and needs to be addressed.

- Open a project and tool.
- Expand/Collapse submenu in the Group Menu.
- CAUTION: Only use headers with single words until data issue is resolved.
- Click on another menu to ensure that it opens as expected.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2490)
